src/commands/ai/extract-structured.js

```diff
@@ -0,0 +1,140 @@
+	'box ai:extract-structured --items="id=12345,type=file" --metadata-template="type=metadata_template,scope=enterprise,template_key=test" --ai-agent \'{"type":"ai_agent_extract","basicText":{"llmEndpointParams":{"type":"openai_params","frequencyPenalty": 1.5,"presencePenalty": 1.5,"stop": "<|im_end|>","temperature": 0,"topP": 1},"model": "azure__openai__gpt_4o_mini","numTokensForCompletion": 8400,"promptTemplate": "It is, consider these travel options and answer the.","systemMessage": "You are a helpful travel assistant specialized in budget travel"},"longText":{"embeddings":{ "model": "azure__openai__text_embedding_ada_002","strategy":{"id": "basic","numTokensPerChunk": 64}},"llmEndpointParams":{"type":"openai_params","frequencyPenalty": 1.5,"presencePenalty": 1.5,"stop": "<|im_end|>","temperature": 0,"topP": 1},"model":"azure__openai__gpt_4o_mini","numTokensForCompletion":8400,"promptTemplate":"It is , consider these travel options and answer the.","systemMessage":"You are a helpful travel assistant specialized in budget travel"}}\'',
+];
+AiExtractStructuredCommand._endpoint = 'post_ai_extract_structured';
+
+AiExtractStructuredCommand.flags = {
+	...BoxCommand.flags,
+	items: Flags.string({
+		required: true,
+		description: 'The items for the AI request (id, type, optional content)',
+		multiple: true,
+		parse(input) {
+			const item = {
+				id: '',
+				type: 'file',
+			};
+			const obj = utils.parseStringToObject(input, ['id', 'type', 'content']);
+			for (const key in obj) {
+				if (key === 'id') {
+					item.id = obj[key];
+				} else if (key === 'type') {
+					item.type = obj[key];
+				} else if (key === 'content') {
+					item.content = obj[key];
+				} else {
+					throw new Error(`Invalid item key ${key}`);
+				}
+			}
+			if (!item.id) {
+				throw new Error('Item must include an id');
+			}
+			return item;
+		},
+	}),
+	'metadata-template': Flags.string({
+		description: 'metadata template to use for the AI request',
```

Comment:
Upper case for Metadata.

For most of the description, you should reuse the description we have from the Box API: 
<img width="731" alt="image" src="https://github.com/user-attachments/assets/3021f2dc-42ad-4dbb-a129-a3d1d706d0ed" />

And you can let user know which field is able to put into this metadata template field.